roles/parallels/tasks/main.yml

```diff
@@ -30,13 +30,47 @@
 # 4C6364ACXT|com.parallels.kext.hypervisor|1|Parallels International GmbH|1
 # 4C6364ACXT|com.parallels.kext.usbconnect|1|Parallels International GmbH|1
 #
-- name: Check if Paralles kernel extensions are approved
+- name: Check if Parallels kernel extensions are approved
   command: sqlite3 /var/db/SystemPolicyConfiguration/KextPolicy 'SELECT COUNT(*) from kext_policy WHERE team_id = "4C6364ACXT";'
   register: kernel_extension_policy
   changed_when: no
   tags:
     - parallels
     - parallels_check
+  when:
+    - parallels_assert_kexts
+
+- name: Pause to give a chance to approve the kernel modules via GUI
+  ansible.builtin.pause:
```

Comment:
nit: This could just be `pause`.

Reply:
Yeah.. I got into a habit of being explicit.

Reply:
I won't hold it against you. 😉